tools/run_tests/performance/build_performance.sh

```diff
@@ -43,6 +43,9 @@ do
   "go")
     tools/run_tests/performance/build_performance_go.sh
     ;;
+  "php")
+    tools/run_tests/performance/build_performance_php.sh
```

Comment:
I'll change it to `tools/run_tests/performance/build_performance_php.sh CONFIG`.
It runs well currently.

Reply:
Done.